docs/release_notes.rst

```diff
@@ -52,6 +51,16 @@ contains valid data (i.e., using type-specific getters, or obtaining the entire
 sample as an object). When the instance state is NOT_ALIVE_DISPOSED, only the
 key values in the sample should be accessed.
 
+[RTI Issue ID CON-188]
+
+Support for loading additional libraries
+""""""""""""""""""""""""""""""""""""""""
+
+It is now possible to load additional Connext DDS libraries at runtime. This means
+that Connext DDS features such as Monitoring and Security Plugins are now supported.
+Refer to :ref:`Loading additional Connext DDS Libraries` for more information.
+
+[RTI Issue ID CON-221]
 
```

Comment:
For new features, we do not display the RTI Issue ID. Only for fixes do we display the Issue ID at the end.